### DISCUSSION TITLE
Keep Batect and your configuration up-to-date automatically with Renovate

### DISCUSSION THREAD

**Discussion by charleskorn** [batect/batect#767] [Announcements]
_tl;dr: [Renovate](https://renovatebot.com/) now supports Batect, which means it can automatically send you PRs to update Batect itself as well as any images or bundles you reference in your configuration files._

Keeping dependencies and tools up-to-date is a necessary but tedious task for any codebase. Tools like [Dependabot](https://dependabot.com/) and [Renovate](https://renovatebot.com) help to automate this process by scanning your repository for dependencies, identifying which ones are out of date and then sending you PRs to update them.

I'm pleased to announce that Renovate has accepted a series of PRs that add support for Batect 🎉 

Read more on [the blog](https://batect.dev/blog/2020/12/20/renovate-support/).

**Comment by sschuberth** [comment]
@charleskorn, could you please also explain how to *disable* the Renovate check for Batect at https://batect.dev/docs/using-batect-with/tools/renovate/? I tried

```
   "extends": [
     "batect:disable"
   ],
```

but that does not work (Renovate complains about invalid syntax).

**Reply by sschuberth** [reply]
That is, I do want to get upgrades to Batect / the wrapper itself, but I do not want to get upgrades to Docker base image I use in `batect.yml` files.

**Reply by sschuberth** [reply]
BTW, I used `batect:disable` because `docker:disable` works for Dockerfiles. But I guess the part before ":" is supposed to be a "programming lanugage" and not a "manager", and the "docker" progrgamming language maps to the [managers](https://docs.renovatebot.com/modules/manager/) `ansible`, `docker-compose`, `dockerfile`, `droneci`, `gitlabci`, `kubernetes`, `woodpecker`. Which makes me think, should `batect` be part of that list?

**Reply by sschuberth** [reply]
Anyway, it's pretty clear now that "batect" as a manager can be disabled via

```
{
  "batect": {
    "enabled": false
  }
}
```

**Reply by charleskorn** [reply]
This is probably a question best asked in https://github.com/renovatebot/renovate/discussions.

As far as I'm aware, `extends` refers to presets, and I believe presets are created completely separately to the managers themselves. It looks like the `docker:disable` preset was created specifically.



> Anyway, it's pretty clear now that "batect" as a manager can be disabled via
> 
> ```
> {
>   "batect": {
>     "enabled": false
>   }
> }
> ```

That looks right to me based on my experience with Renovate.

**Reply by sschuberth** [reply]
> `extends` refers to presets

Yes, right, that's what I learned by now, too. It's about presets, not programming languages.
